examples/common/profile_sla.py

```
@@ -474,6 +487,14 @@ def benchmark_decode(isl, osl, num_request, genai_perf_artifact_dir, model_name,
     )
     args = parser.parse_args()
 
+    if args.example_dir is None:
+        logger.info("Example directory not provided, inferring from config file location...")
+        try:
+            args.example_dir = os.path.dirname(os.path.dirname(args.config))
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)